swagger.yaml

```diff
@@ -2202,3 +2208,15 @@ definitions:
         format: ipv4
         description: The IPv4 address that was allocated by DHCP for Ingress VIP.
 
+  cluster-validation-id:
+    type: string
+    enum:
+      - 'machine-cidr-defined'
+      - 'machine-cidr-equals-to-calculated-cidr'
+      - 'api-vip-defined'
+      - 'api-vip-valid'
+      - 'ingress-vip-defined'
+      - 'ingress-vip-valid'
+      - 'no-pending-for-input-host'
```

Comment:
Not needed. all-hosts-are-ready-to-install should be enough.
In addition, I think the user will be confused between these two validations.